docs/src/security/transparency/2021_report.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "2021 Platform.sh Transparency Report"
+sidebarTitle: "2021 Report"
+file: "/files/reports/transparency-abuse/2021_platformsh_transparency_report.pdf"
```

Comment:
The file also includes the typos:
- `maleware` -> **malware**
- `infingement` -> **infringement**